packages/good-design/src/advanced/web3action/Web3Action.tsx

```diff
@@ -31,7 +31,7 @@ const StepIndicator: FC<{ text?: string } & ITextProps> = withTheme({ name: "Ste
   ({ text, color, fontSize }) => (
     <HStack space={2} alignItems="center" flexDirection="row">
       <Spinner accessibilityLabel="Waiting on wallet confirmation" />
-      <Heading color={color} fontSize={fontSize}>
+      <Heading color={color} fontSize={fontSize} fontFamily='Montserrat'>
```

Comment:
should be semantic ('body', 'heading' etc). fix it **everywhere**

Reply:
@johnsmith-gooddollar it was added because of issue that I told you yesterday, in case with alias as 'heading' it's not applying correct font on web, because it takes value from fontsConfig, which not exist on web fonts (e.g. 'Montserrat-Bold')

Reply:
@alt-a1r again I repeat - fonts config is **just a web to native mapping**
it ignored on web

- when you use 'Montserrat' on web it should render as CSS `font-face: 'Montserrat'` and set weight / style you've specified
- when you use 'Montserrat' on native it looks up by this name as key in fontsConfig object, then takes weight / style values and picks up the proper full font name to use

Reply:
@alt-a1r if something goes wrong that means
a) error in native base try upgrade/downgrade
b) error in helmet - fonts not loaded
c) error in build settings - for some reason native base determines env incorrectly and applies mapping thinking it's ran on native (when running actually in the browser)

Reply:
@johnsmith-gooddollar 
Whats happening is, that in current setup its based on a 'custom font family' meaning it needs the associated font to work.

fontConfig normal/italic names is what is being applied as font-family in css

There is nothing mapping Montserrat-Bold > Montserrat, except when we are generating the google-api URL
The font is not being loaded because if this is the URL generated:
[https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0[…]200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap](https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap)

Then the fontFamily applied to the component should be Montserrat, not Montserrat-Bold which is whats happening now. Montessart-Bold doesn't exist in any way in this context, it would not make a proper working-url


the implementation as is with fontConfig relies on this being done:
https://aravindmnair.medium.com/add-custom-fonts-to-react-native-0-60-easily-in-3-steps-fcd71459f4c9

Actual custom fonts.

`<Text fontFamily="body" fontWeight="600" fontStyle="italic" />`
where fontWeight here does not apply to actual fontWeight, but to which variant out of the font-config it loads

Reply:
@L03TJ3 please read my post and stop arguing
I'm trying on bare project now. suggest you to do the same

Reply:
`<Text fontFamily="body" fontWeight="100" style={{fontWeight: "100"}} fontStyle="italic />`

This is the only thing that currently would work, but only if the name of the 'body' associated font 'Montserrat' is actually 'Montserrat'

```
export const fontConfig = {
  Montserrat: {
    100: {
      normal: "Montserrat",
      italic: "Montserrat",
    },
        200: {
      normal: "Montserrat-ExtraLight",
      italic: "Montserrat-ExtraLightItalic",
    },`
```

Where now it would load in the italic 100

applying fontWeight prop (not style fontWeight) 200 in this example would not load montserrat